client/js/components/user/DpOrganisationList/DpOrganisationFormFields.vue

```diff
@@ -177,6 +177,18 @@
         </p>
       </div>
 
+      <addon-wrapper
+        hook-name="mein.berlin.additional.field"
+        :addon-props="{
+          class: 'ml-4',
+          relationshipId: this.organisationId,
+          resourceType: 'MeinBerlinAddonOrganisation',
+          label: Translator.trans('organisation.mein.berlin.id'),
```

Comment:
Addon names and specifics should not be exposed in core

Reply:
yes, I got it and moved it to the addon. is it ok now? https://github.com/demos-europe/demosplan-core/pull/3999/files#diff-972e4459afe8be8598e4910ee7db76f0eee580c7c0ee7d8ce78224f0593230edR180-R190

Reply:
Thank you, this is way better. I think in the long run we need some kind of hook here that multiple addons could add their fields here but for now this is probably enough